tower_cli/utils/parser.py

```diff
@@ -127,7 +127,7 @@ def process_extra_vars(extra_vars_list, force_json=True):
             opt_dict = string_to_dict(extra_vars_opt, allow_kv=False)
         else:
             # Convert text markup to a dictionary liberally
-            opt_dict = string_to_dict(extra_vars_opt, allow_kv=True)
+            opt_dict = string_to_dict(extra_vars_list, allow_kv=True)
```

Comment:
Checks seem to fail in travis, but I am not sure if that's because my fix is bad. I am not sure if this line has to be in for loop. I think if we are passing entire list, it can be out of the `for` loop.